purchase_fop_shipping/tests/test_fop_shipping.py

```diff
@@ -0,0 +1,87 @@
+# © 2017 Akretion (http://www.akretion.com)
+#   @author Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields
+from odoo.exceptions import UserError
+from odoo.tests import tagged
+from odoo.tests.common import TransactionCase
+
+
+@tagged("post_install", "-at_install")
+class TestPurchaseOrder(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.Product = cls.env["product.product"]
+        cls.Purchase = cls.env["purchase.order"]
+        cls.PurchaseLine = cls.env["purchase.order.line"]
+
+        cls.product_1 = cls.Product.create({"name": "Product", "type": "consu"})
```

Comment:
@dzungtran89 I think I should remove `at_install` to avoid this error situation when create product. Because this field `sale_line_warn` is added on module `sale`, which my migrate module does not depend on module "sale".
<img width="1383" alt="Ảnh chụp Màn hình 2024-07-05 lúc 15 28 36" src="https://github.com/traitrantrobz/purchase-workflow/assets/144201067/e09ee953-205d-4810-a236-440752e3968d">